digitalocean/kubernetes/kubernetes.go

```diff
@@ -240,6 +240,19 @@ func expandMaintPolicyOpts(config []interface{}) (*godo.KubernetesMaintenancePol
 	return maintPolicy, nil
 }
 
+func expandControlPlaneFirewallOpts(raw []interface{}) *godo.KubernetesControlPlaneFirewall {
+	if len(raw) == 0 || raw[0] == nil {
+		return &godo.KubernetesControlPlaneFirewall{}
+	}
+	controlPlaneFirewallConfig := raw[0].(map[string]interface{})
+
+	controlPlaneFirewall := &godo.KubernetesControlPlaneFirewall{
+		Enabled:          controlPlaneFirewallConfig["enabled"].(*bool),
```

Comment:
It may not be able to cast to the pointer directly. What would you say if we cast it to bool first and then pass the pointer?

```suggestion
		Enabled:          godo.PtrTo(controlPlaneFirewallConfig["enabled"].(bool)),
```